libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -460,6 +465,15 @@ bool AP_Arming::pre_arm_checks(bool report)
     return ret;
 }
 
+bool AP_Arming::arm_checks(uint8_t method)
+{
+    if (!DataFlash_Class::instance()->logging_started()) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_CRITICAL, "Arm: Logging not started");
```

Comment:
Shouldn't this only be sent if we want logging as part of the arming check? If someone opts out of the logging arming check I can see that sending this would be annoying/confusing.